### PR TITLE
1.2.1 mainnet deployment [PE-1433]

### PR DESCRIPTION
**Pull request by iamsahu**
## Description

This PR deploys the 1.2.1 contracts to mainnet.

## Notice

- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/liquid-collective/liquid-collective-protocol/pulls) for the same update/change?
- [x] Have you assigned this PR to yourself?
- [x] Have you added at least 1 reviewer?
- [x] Have you updated the official documentation?
- [x] Have you added sufficient documentation in your code?
- [ ] Have you added relevant tests to the official test suite?

## Pull Request Type

- [ ] 💫 New Feature (Breaking Change)
- [ ] 💫 New Feature (Non-breaking Change)
- [ ] 🛠️ Bug fix (Non-breaking Change: Fixes an issue)
- [x] 🕹️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)

## Testing

- [x] Have you tested this code with the official test suite?
- [ ] Have you tested this code manually?

### CODE REVIEW

File: deploy/mainnet/11_deploy_v1.0.1.ts [hunk 1]
```diff
@@ -9,17 +9,17 @@ const func: DeployFunction = async function ({ deployments, network, getNamedAcc
...
-  await deployments.deploy("OperatorsRegistryV1_Implementation_1_0_1", {
-    contract: "OperatorsRegistryV1",
-    from: deployer,
-    log: true,
-  });
+  // await deployments.deploy("OperatorsRegistryV1_Implementation_1_0_1", {
+  //   contract: "OperatorsRegistryV1",
+  //   from: deployer,
+  //   log: true,
+  // });
```
Comment (koderholic) [inline deploy/mainnet/11_deploy_v1.0.1.ts:22]: Same here

File: deploy/mainnet/13_deploy_v1_1_1.ts [hunk 2]
```diff
@@ -16,12 +16,12 @@ const func: DeployFunction = async function ({
...
-    from: deployer,
-    log: true,
-  });
-  console.log(allowlistNewImplementationDeployment.address);
+  // const allowlistNewImplementationDeployment = await deployments.deploy("AllowlistV1_Implementation_1_1_1", {
+  //   contract: "AllowlistV1",
+  //   from: deployer,
+  //   log: true,
+  // });
+  // console.log(allowlistNewImplementationDeployment.address);
```
Comment (koderholic) [inline deploy/mainnet/13_deploy_v1_1_1.ts:24]: Same here.

File: deploy/mainnet/04_deploy_wlseth.ts [hunk 3]
```diff
@@ -7,20 +7,20 @@ const func: DeployFunction = async function ({ deployments, getNamedAccounts }:
...
+  //   proxy: {
+  //     implementationName: "WLSETHV1_Implementation_0_2_2",
+  //     owner: proxyAdministrator,
+  //     proxyContract: "TUPProxy",
+  //     execute: {
+  //       methodName: "initWLSETHV1",
+  //       args: [riverDeployment.address],
+  //     },
+  //   },
+  // });
```
Comment (koderholic) [inline deploy/mainnet/04_deploy_wlseth.ts:23]: With tags we would not need to comment this out.

File: deploy/mainnet/11_deploy_v1.0.1.ts (hunk 1 above)
Comment (iamsahu) [reply deploy/mainnet/11_deploy_v1.0.1.ts:22]: Currently tags are not part of the lc repo. Can you please create another PR to introduce the tag system for deployment in LC as well?

**Review by juliaaschmidt — APPROVED** [review]
LGTM

**Review by koderholic — APPROVED** [review]
LGTM

**Review by mischat — APPROVED** [review]
LGTM

**Pull request merged**